blog/_posts/2020-1-23-crash-course-kubernetes-nodes.md

```diff
@@ -0,0 +1,168 @@
+
+Hello and welcome to our now slightly less-new blog post series, “A Crash Course on Kubernetes!” 
+
+Our summer intern didn’t have a background in Kubernetes, so to get her up to speed, we started
+giving internal Tilt University presentations on relevant concepts. This was a great crash
+course for our intern, but even our experienced engineers learned things at these talks--turns
+out, you can work within Kubernetes pretty effectively and still have large gaps in your
+mental models.
+
+We’re blogifying our Tilt U presentations so that other folks can benefit from the digging we did.
+This is post #2 of the series, in which we talk about nodes, a.k.a "the things that hold the things
+that run your app."
+
+If you missed [Post 1: Overview](https://blog.tilt.dev/2019/10/16/crash-course-kubernetes
+-overview.html), you might want to start there. Stay tuned for post #3, which will cover pods
+(the things that actually run your app) and services (how pods talk to each other and the
+outside world).
+
+## What is a Node, and Why Should I Care?
+In the above diagram, the blue squares are nodes. A node is just a machine (physical or virtual,
+it doesn’t matter) that runs some containers, and a Kubernetes process or two. There are two
+types of nodes: **worker nodes**, where your code runs, and the **master node**, which is in
+charge of managing the cluster.
+
+Like most other parts of Kubernetes, nodes are easy to ignore and abstract away when your
+cluster is in a happy state. When things start to break, however, it can be really useful to
+understand just what Kubernetes is doing. 
+
+If you start getting errors like “node(s) had taints” and “could not schedule pod”, knowing how
+nodes work, and how pods are assigned to nodes, will help you figure out what’s going on. And
+if things start to go wrong (“Unable to connect to the [API] server”), or if your `kubectl
+apply` didn’t work like you expected it to, an accurate mental model of the master node and
+all the machinery that runs your cluster will be a lifesaver.
+
+### Wait, What’s a Pod?
+Simply put, a pod is one unit of your app---say, a running instance of `shopping_cart_server`.
+There’s a bit more to it than that, of course, but we’ll delve into the intricacies of pods in
+the next post in this series. 
+
+For the discussion of nodes, all you need to know is that a pod runs your app in a container,
+and pods themselves are containers that run on nodes. 
+
+## Anatomy of a Node
+### Master Node
+First we’ll talk about the master node. It’s called the “master node” because it’s in charge of everything.
+
+In Kubernetes, you specify your configuration _declaratively_---you tell Kubernetes the desired
+state of your cluster (number of instances of your app, which scaling method to use, how to set
+up the networking) and Kubernetes makes it so. 
+
+The master node is how: it is both the means by which you specify your desired state, and the
+machinery for making sure that state is maintained.
```

Comment:
```suggestion
machinery for making sure the specified state is obtained.
```

or a different wording, but it's not just about preserving state, but transforming to new states